lib/Connector/XiboAudienceReportingConnector.php

```diff
@@ -192,6 +193,9 @@ public function onRegularMaintenance(MaintenanceRegularEvent $event)
                     $entry['campaignEnd'] = $campaignCache[$parentCampaignId]['end'];
                 } else {
                     $parentCampaign = $this->campaignFactory->getById($parentCampaignId);
+                    if ($parentCampaign->type !== 'ad') {
```

Comment:
I think we should cache the result of this lookup, otherwise a query returning 10000 stats all for layout list campaigns will make the lookup 10000 times?